File: tests/fixtures/ts/destructured/src/billing.ts
import Stripe from 'stripe';

const stripe = new Stripe(process.env.STRIPE_KEY!);

export async function charge(amount: number) {
  const { id, status: chargeStatus } = await stripe.charges.create({ amount, currency: 'usd' });
  return { id, chargeStatus };
}

File: tests/fixtures/py/unicode/src/billing.py
from stripe import StripeClient

"""Facturación para la región de España — cobros en euros."""

client = StripeClient("sk_test")


def cobrar(importe: int):
    # Créer une charge: le montant est en centimes (déjà converti).
    resultado = client.charges.create(amount=importe, currency="eur")
    return resultado.status
